tests/indexers/offset_tracker/test_rated.py

```diff
@@ -35,6 +42,37 @@ def yaml_config():
     return config
 
 
+@pytest.fixture
+def yaml_config_with_customer():
+    slaos_config = OffsetSlaosYamlConfig(
+        ingestion_id=INGESTION_ID,
+        ingestion_key=INGESTION_KEY,
+        ingestion_url=INGESTION_URL,
+        datastream_filter=OffsetSlaosYamlFilter(
+            key=DATASTREAM_KEY, customer_id=CUSTOMER_ID
+        ),
+    )
+    config = OffsetYamlConfig(
+        type=OffsetTypes.SLAOS,
+        override_start_from=False,
+        start_from=3456,
+        start_from_type=StartFromTypes.BIGINT,
+        slaos=slaos_config,
+    )
+    return config
+
+
+@pytest.fixture
+def mock_http_client():
+    with unittest.mock.patch("httpx.Client") as mock_client:
+        mock_response = unittest.mock.Mock()
+        mock_response.raise_for_status = lambda: None
+        mock_response.json.return_value = {}
+
+        mock_client.return_value.get.return_value = mock_response
+        yield mock_client
```

Comment:
Keep it simple. In the scope of the offset tracker module we don't need to concern ourselves with HTTP, so no need to create a mock for this.  Just patch `get_offset_from_api`, set a return value on that and if you want check what it was called with. HTTP is a concern of the `SlaosClient` which is out of scope here, so we don't even want to invoke it.